pkg/gardenlet/controller/seed/care/health.go

```diff
@@ -156,12 +157,12 @@ func (h *SeedHealth) checkSeedSystemComponents(
 	return &c, nil
 }
 
-func checkManagedResourceForSeed(checker *HealthChecker, condition gardencorev1beta1.Condition, managedResource *resourcesv1alpha1.ManagedResource) *gardencorev1beta1.Condition {
+func checkManagedResourceForSeed(checker *care.HealthChecker, condition gardencorev1beta1.Condition, managedResource *resourcesv1alpha1.ManagedResource, clock clock.Clock) *gardencorev1beta1.Condition {
```

Comment:
As you have removed the specific parts at other places, do you also want to drop it here?

```suggestion
func checkManagedResource(checker *care.HealthChecker, condition gardencorev1beta1.Condition, managedResource *resourcesv1alpha1.ManagedResource, clock clock.Clock) *gardencorev1beta1.Condition {
```

Reply:
See https://github.com/gardener/gardener/pull/8378#discussion_r1303137677.